<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perimeter Manager</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QComboBox" name="comboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>Perimeter_Samstagern_1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perimeter_Samstagern_2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perimeter_Schwyz_1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perimeter_Schwyz_2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Perimeter_KT_ZH</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="error_count_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="flash_button">
        <property name="toolTip">
         <string>Highlight current feature on map</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="auto_pan_button">
        <property name="toolTip">
         <string>Automatically pan to the current coordinates / feature</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="auto_zoom_button">
        <property name="toolTip">
         <string>Automatically zoom to the current feature / coordinates</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Perimeter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QWidget" name="validation_layout" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QProgressBar" name="progress_bar">
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <row>
          <property name="text">
           <string>Normschacht 125 - imported bereits mit Perimeter_Schwyz_1 exportiert</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../qgis/QGIS/images/themes/default/mIconWarning.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mIconWarning.svg</iconset>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Einleitstelle 12 b - 002  bereits mit Perimeter_Schwyz_1 exportiert</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../qgis/QGIS/images/themes/default/mIconWarning.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mIconWarning.svg</iconset>
          </property>
         </row>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="result_table_view"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QToolButton" name="export_2">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/export_perimeter.svg</normaloff>../images/export_perimeter.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="import_2">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/import_perimeter.svg</normaloff>../images/import_perimeter.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/xtfexport.svg</normaloff>../images/xtfexport.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../images/xtfimport.svg</normaloff>../images/xtfimport.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_3">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>299</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="project">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../qgis/QGIS/images/themes/default/mIconQgsProjectFile.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mIconQgsProjectFile.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="selectintersection">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../qgis/QGIS/images/themes/default/mActionSelectPolygon.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mActionSelectPolygon.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="link">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../qgis/QGIS/images/themes/default/mActionLink.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mActionLink.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="unlink">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../qgis/QGIS/images/themes/default/mActionUnlink.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mActionUnlink.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="highlight">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../qgis/QGIS/images/themes/default/mActionHighlightFeature.svg</normaloff>../../../../qgis/QGIS/images/themes/default/mActionHighlightFeature.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
